src/Sarif.Multitool/Rules/SARIF2004.OptimizeFileSize.cs

```diff
@@ -69,10 +77,46 @@ private bool HasResultLocationsWithUriAndIndex(string resultPointer)
 
         private bool HasLocationOnlyArtifacts(string artifactPointer)
         {
-            JToken artifactToken = artifactPointer.ToJToken(Context.InputLogToken);
+            var artifactToken = artifactPointer.ToJToken(Context.InputLogToken);
```

Comment:
>var [](start = 12, length = 3)

Revert. Even though the name `ToJToken` is quite clear, still our standard is only to use `var` when the right-hand side is `new T`, `(T)expr`, or `expr as T`; the type must be absolutely explicit.

Reply:
Okay :) 

---
In reply to: [447982484](https://github.com/microsoft/sarif-sdk/pull/1950#discussion_r447982484) [](ancestors = 447982484)